### PR TITLE
[MU3] Sync CI build scripts with master

### PR DESCRIPTION
**Pull request by igorkorsukov**
Sync CI build scripts with master

### CODE REVIEW

File: .github/workflows/ci_linux.yml [hunk 1]
```diff
@@ -31,7 +31,8 @@ jobs:
       run: |
         sudo bash ./build/ci/tools/make_build_mode_env.sh -e ${{ github.event_name }} -m ${{ github.event.inputs.build_mode }}
         BUILD_MODE=$(cat ./build.artifacts/env/build_mode.env)
-        echo "BUILD_MODE=$BUILD_MODE" >> $GITHUB_ENV
+
+	DO_BUILD='true'
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_linux.yml:35]: Really a Tab here?

File: .github/workflows/ci_macos.yml [hunk 2]
```diff
@@ -61,17 +58,30 @@ jobs:
             DO_NOTARIZE='false'
           fi 
         fi
+
+	DO_BUILD='true'
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_macos.yml:62]: same

File: .github/workflows/ci_windows.yml [hunk 3]
```diff
@@ -44,29 +43,39 @@ jobs:
...
 
         DO_CHECKSUM='true'
         if [ "$BUILD_MODE" == "devel_build" ]; then DO_CHECKSUM='false'; fi
-        echo "DO_CHECKSUM=$DO_CHECKSUM" >> $GITHUB_ENV
-        echo "DO_CHECKSUM: $DO_CHECKSUM"
 
         DO_APPCAST='true'
         if [ "$BUILD_MODE" == "devel_build" ]; then DO_APPCAST='false'; fi
+
+	DO_BUILD='true'
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_windows.yml:53]: here too

File: .github/workflows/ci_linux.yml (hunk 1 above)
Comment (igorkorsukov) [reply .github/workflows/ci_linux.yml]: fixed, thank you!

File: .github/workflows/ci_macos.yml (hunk 2 above)
Comment (igorkorsukov) [reply .github/workflows/ci_macos.yml]: fixed, thank you!

File: .github/workflows/ci_windows.yml (hunk 3 above)
Comment (igorkorsukov) [reply .github/workflows/ci_windows.yml]: fixed, thank you!

**Pull request merged**